app/assets/javascripts/bus_stops.js

```diff
@@ -2,6 +2,10 @@ $(document).ready(function(){
   $('.name-autocomplete').autocomplete({
     source: searchStops
   });
+
+  $('.datepick').datepicker({
+    dateFormat: 'yy-mm-dd'
```

Comment:
It *looks* like a two-digit year, but it's actually four. If you write `yyyy` you end up with `20182018`.

Reply:
Waaaaaat